src/App.svelte

```diff
@@ -88,20 +88,26 @@
 		{ display: "FAQ", action: () => scrollToID("faq")},
 		{ display: "SPONSORS", action: () => scrollToID("sponsors")},
 		{ display: "PRIZES", action: () => scrollToID("prizes")},
-		{ display: "MEET the TEAM", action: () => scrollToID("contact")},
-		{ display: "HACKERDOC", action: () => window.open("https://hackku.notion.site/hackku/HackerDoc-HackKU-2023-d870cdb8e84b425ab67a2eedcb41344c", "_blank")},
-		{ display: "REGISTER NOW!", action: () => window.open("https://forms.gle/KwRNWYcxXyqf3EdZ7", "_blank")}
+		{ display: "MEET the TEAM", action: () => scrollToID("contact")}
```

Comment:
should not have gotten rid of "hackerdoc" and "register now".

Reply:
HackerDoc and register now are hardcoded as <li> elements so i thought it would be better to leave it untouched.

Reply:
this is still used for the mobile nav, so hackerdoc and register now need to remain here as they are

Reply:
Ohhhhhhhh i see, my bad ill fix the changes then

Reply:
for mobile, change order of links to

1. register now
2. about
3. faq
4. sponsors
5. prizes
6. meet the team
7. hackerdoc